miniapp/miniapp.cpp

```diff
@@ -224,7 +224,7 @@ std::unique_ptr<sample_trace_type> make_trace(cell_member_type probe_id, probe_s
     }
     name += probe.location.segment? "dend" : "soma";
 
-    return util::make_unique<sample_trace_type>(probe_id, name, units);
+    return nest::mc::util::make_unique<sample_trace_type>(probe_id, name, units);
```

Comment:
It isn't necessary to use the full namespace `nest::mc::` is it? This is very verbose.